File: NotificaCrimesBackEnd/AppNotificacoesCrimesCidade.Domain/Entities/Roubo.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AppNotificacoesCrimesCidade.Domain.Entities
{
    public class Roubo
    {
        [Column("id")]
        public int Id { get; set; }

        [Column("id_ocorrencia")]
        public int OcorrenciaId { get; set; }

        public Ocorrencia Ocorrencia { get; set; }

        [Column("tentativa")]
        public bool Tentativa { get; set; }

        public List<RouboTipoBem> RouboTipoBens { get; set; }
    }
}

File: notifica_crimes_frontend/lib/domain/models/ocorrencias/roubo.dart
import 'package:notifica_crimes_frontend/domain/models/ocorrencias/ocorrencia.dart';

class Roubo {

  final bool tentativa;
  final List<String> tipoBensId;
  final Ocorrencia ocorrencia;

  Roubo({required this.tentativa, required this.tipoBensId, required this.ocorrencia, });

}